<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfForm</class>
 <widget class="QDialog" name="ConfForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="stashDirectoryLabel">
       <property name="text">
        <string>Project Dir:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="projectEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="openButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="defaultFileNameLabel">
       <property name="text">
        <string>Default Files:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="SLineEdit">
         <property name="text">
          <string>Solver.cpp</string>
         </property>
         <property name="placeholderText">
          <string notr="true">Solver</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="maxBitsLabel">
       <property name="text">
        <string>Table:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSpinBox" name="bytesPerCellSpinBox">
         <property name="suffix">
          <string> bytes/cell</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100001</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="rowsPerPageSpinBox">
         <property name="suffix">
          <string notr="true"> rows/page</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100001</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="initialRowsLabel">
       <property name="text">
        <string>File Viewer:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="bytesPerPageSpinBox">
       <property name="suffix">
        <string notr="true"> bytes/page</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100001</number>
       </property>
       <property name="value">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="pipeLabel">
       <property name="text">
        <string>Pipe:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="bytesPerReadSpinBox">
       <property name="suffix">
        <string notr="true"> bytes/read</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000001</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="buildOptsLabel">
       <property name="text">
        <string>Build Mode:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QGroupBox" name="buildModeGroupBox">
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>-1</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="buildDebugRadioButton">
          <property name="text">
           <string>Debug</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="buildReleaseRadioButton">
          <property name="text">
           <string>Release</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="buildAsNeedCheckBox">
       <property name="text">
        <string>Build as need</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QCheckBox" name="runinshellCheckBox">
       <property name="text">
        <string>Run in shell mode</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="dumpWarmUpLabel">
       <property name="text">
        <string>Dump/Warm:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="exeDumpSpinBox">
         <property name="suffix">
          <string notr="true"> seconds</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="exeWarmSpinBox">
         <property name="suffix">
          <string notr="true"> seconds</string>
         </property>
         <property name="minimum">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="parallelLabel">
       <property name="text">
        <string>Parallels:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QSpinBox" name="parallelSpinBox">
         <property name="suffix">
          <string notr="true"> threads</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>32</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="refreshRateSpinBox">
         <property name="suffix">
          <string notr="true"> Hz</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>60</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="fontLabel">
       <property name="text">
        <string>Font:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="fontLineEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="fontButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
